src/Chapter2.hs

```diff
@@ -621,7 +627,13 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven l = go True l
+  where
+    go :: Bool -> [a] -> [a]
+    go _ [] = []
+    go True (x:xs) = x : go False xs
+    go False (_:xs) = go True xs
```

Comment:
Nice idea on the helper functions :+1:

Alternatively, you could use pattern matching in `takeEven` instead. As you need to take every second element, it is possible to represent with a few pattern matching clauses 🙂